scormxblock/static/js/src/scormxblock.js

```diff
@@ -382,6 +382,36 @@
             initPendingValues();
             window.API = new SCORM_12_API();
             window.API_1484_11 = new SCORM_2004_API();
+
+            $('.launch-button', element).on('click', function() {
+                window.API = new SCORM_12_API();
```

Comment:
- As we share `window.API` between multiple SCORM instances in one page. So we recreate API object while using